liqd_product/assets/scss/components/_cms.scss

```diff
@@ -13,12 +12,30 @@
     font-weight: 600;
```

Comment:
I know, this was here before, but we should probably define a variable for that? demibold?